function.js

```diff
@@ -0,0 +1,10 @@
+'use strict';
+
+module.exports = {
+	getId
+};
+
+function getId(userContext, events, done) {
+	userContext.vars.id = Math.floor(Math.random() * 100000);
```

Comment:
@kchen395, did you want the 'id' to only be picked from 1 to 100000? Also, be careful with Math.floor because it can give you 0, which I'm assuming isn't in your database.

Reply:
Good catch on Math.floor, I set it to 100000 due to storage limitations on my computer